tests/scripts/combined-coverage.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Run unit test and visual coverage test by default.
+# Skip this step if merge-only flag is provided.
+echo $1
+if [ "$1" != "merge-only" ]
```

Comment:
Was merge-only for testing purposes?

Reply:
ah yes, for testing purposes. I was experimenting with avoiding reruns in github actions. The upload/download artifacts took care of it. This flag is kinda nice when you are checking coverage but don't want to rerun things, although you can argue they can just comment out those two lines. I can remove it

Reply:
I will just remove it actually

Reply:
sure I don't mind keeping it was just wondering